govuk_sidekiq.gemspec

```
@@ -21,8 +21,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "gds-api-adapters", ">= 19.1.0"
   spec.add_dependency "govuk_app_config", ">= 1.1"
   spec.add_dependency "redis-namespace", "~> 1.6"
-  spec.add_dependency "sidekiq", ">= 5", "< 6"
-  spec.add_dependency "sidekiq-logging-json", "~> 0.0"
+  spec.add_dependency "sidekiq", ">= 6", "< 7"
```

```suggestion
  spec.add_dependency "sidekiq", "~> 6"
```

We should change this to use the [twiddle-wacka](https://guides.rubygems.org/patterns/) as I've found that the previous constraint of `">= 5", "< 6"` was causing dependabot to try update apps to Sidekiq 6.0.0.pre1 ([example](https://github.com/alphagov/short-url-manager/pull/783/commits)) which is annoying. I'm not sure but I think the `">= 6", "< 7"` constraint may mean apps get offered "7.0.0.beta1" however that can't seem to happen with twiddle-wacka.

---

updated this, I know this profession has a whole host of stupid names for things but I think twiddle-wacka may very well take the biscuit...

---

Did you mean to update this to `~> 6`?

---

yeah, that's in the gemspec already